### DISCUSSION TITLE
how to solve error: ErrImagePull

### DISCUSSION THREAD

**Discussion by yugasun** [kubevela/kubevela#5071] [Q&A]
When I run `vela addon enable fluxcd`, the process will always be timeout because error `ErrImagePull`, how can I accelerate pulling mirror?

**Comment by wonderflow** [comment]
It's because of the network error or the firewall, you can find some alternative image registry. You can modify the addon locally and modify the image.  https://kubevela.net/docs/platform-engineers/system-operation/enable-addon-offline
